server/config/config.js

```diff
@@ -1,20 +1,15 @@
+const dbConfig = {
+  username: process.env.POSTGRES_USERNAME,
+  password: process.env.POSTGRES_PASSWORD,
+  database: process.env.POSTGRES_DATABASE,
+  host: process.env.POSTGRES_HOST,
+  port: process.env.POSTGRES_PORT,
+  dialect: 'postgres'
+}
 const config = {
-  production: {
-    username: process.env.POSTGRES_USERNAME,
-    password: process.env.POSTGRES_PASSWORD,
-    database: 'mine_users',
-    host: 'mine-support-postgres-users',
-    port: 5432,
-    dialect: 'postgres'
-  },
-  development: {
-    username: process.env.POSTGRES_USERNAME,
-    password: process.env.POSTGRES_PASSWORD,
-    database: 'mine_users',
-    host: 'mine-support-postgres-users',
-    port: 5432,
-    dialect: 'postgres'
-  }
+  production: dbConfig,
+  development: dbConfig,
+  test: dbConfig

```

Comment:
there are two config files, this one in a config directory and another one alongside the config directory.
We should look at tidyiing this up in the future.

Reply:
A lot of the Sequelize-cli commands are dependent on the config being in this folder so we'd need to consider that when we tidy it up.

Reply:
looks like the path to the config files etc are configured in the `.sequelizerc` file:
```
const path = require('path')

module.exports = {
  "config": path.resolve('./server/config', 'config.js'),
  "models-path": path.resolve('./server/models'),
  "seeders-path": path.resolve('./server/seeders'),
  "migrations-path": path.resolve('./server/migrations')
}
```